examples/employees/main.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"log"
 
-	factorial "github.com/charly3pins/factorial-go"
+	factorial "github.com/arexio/factorial-go"
```

Comment:
we usually order the imports as:
- standard library
- external
- internal

so for this case should be shomething like:
```go
"fmt"
"log"

"golang.org/x/oauth2"

factorial "github.com/arexio/factorial-go"
```

Reply:
Yes I don't know why VSCode is sorting this on this way, I though VSCode is using gofmt under the roof buuut who knows, anyway I changed that manually!

Reply:
You can use VIM then :trollface: https://neovim.io/